include/multipass/utils.h

```diff
@@ -51,8 +52,9 @@ QString make_uuid();
 std::string contents_of(const multipass::Path& file_path);
 bool has_only_digits(const std::string& value);
 void validate_server_address(const std::string& value);
-bool valid_memory_value(const QString& mem_string);
-bool valid_hostname(const QString& name_string);
+optional<long long> in_bytes(const std::string& mem_value);
```

Comment:
We don't need sign, so unsigned long long, or uint64_t ?

Reply:
Well I didn't think we were doing arithmetic on it, just passing it around. But I won't object to keeping it as-is.